Assets/Scripts/Controller/Pickup Pup/PPGameController.cs

```diff
@@ -115,6 +142,9 @@ protected override void fetchReferences()
 		dataController = PPDataController.GetInstance;
 		dataController.SetFilePath(SAVE_FILE_PATH);
 		dataController.LoadGame();
+
+        // Defaults not working for some reason. Help me fix!
```

Comment:
So the issue is I wanted the player to start with 2000 coins (which we should make tunable), which is specified in the Currency class. But when I first loaded the game, Coins was equal to 0. Does that mean the Defaults aren't working?